shell/command.go

```diff
@@ -137,7 +143,7 @@ func (c *Command) Run() (progress.Summary, string, error) {
 			stderrOutput = os.Stderr
 		}
 
-		err = c.ScanStderr(stderr, stderrOutput, errors)
+		err = c.analyser.AnalyseLines(io.TeeReader(stderr, io.MultiWriter(stderrOutput, errors)))
 		if err != nil {
 			clog.Errorf("failed reading stderr from command: %s ; Cause: %s", command, err.Error())
```

Comment:
Maybe vice versa, than implemented. 

Stream error handlers do not cause an abort but errors are logged only so that in any case stderr is fully consumed. 

Setup of an invalid regex does cause an error prior to starting the command.

Reply:
Agree

Reply:
`AnalyseLines` now always consumes all stderr content (unless stderr itself would cause an error)